### ISSUE TITLE
Fedora 39 arm-minimal test failed

### ISSUE THREAD

**Issue by mcattamoredhat** [virt-s1/rhel-edge#4907]
Fedora 39 Daily Compose Test- 2023-10-05 failed testing arm-minimal-f39.

Fedora 39 compose Fedora-39-20231005.n.0
Date: 2023-10-05
Compose URL: https://dl.fedoraproject.org/pub/fedora/linux/development/39
Updates URL: https://dl.fedoraproject.org/pub/fedora/linux/development/39

It seems journald has not persistent logging.

```
TASK [assert] ******************************************************************
fatal: [192.168.100.51]: FAILED! => changed=false 
  assertion: result_list_boots.stdout_lines | length > 1
  evaluated_to: false
  msg: journald hasn't persistent logging
```

**Comment by henrywang** [comment]
Fedora 39 for ARM does not enable persistent log.

**Comment by henrywang** [comment]
Fixed by PR #4946
